app/controllers/api/v1/chapters_controller.rb

```diff
@@ -3,46 +3,77 @@
 module Api
   module V1
     class ChaptersController < ApiController
-      before_action :find_chapter, only: [:show, :changes]
-
       def index
-        @chapters = Chapter.eager(:chapter_note).all
+        cache_key = "_v1_chapters-#{actual_date}-view"
+
+        serialized_result = Rails.cache.fetch(cache_key, expires_at: actual_date.end_of_day) do
+          render_to_string(
```

Comment:
This block basically means we can bypass any database queries when something exists for this rendered view